templates/Web/Pages/Angular.List/src/app/components/AngularList/ListForm/list-form.component.html

```diff
@@ -1,16 +1,16 @@
-﻿<div class="input-group my-3">
+﻿<form (submit)="onAddListItem()" class="input-group my-3">
     <input
       type="text"
+      [ngModelOptions]="{standalone: true}"
```

Comment:
Seems good in this case ngModelOptions is an okay choice. We could have went with defining the name property too.

Reply:
I added `name` attribute to the input tag. Though it worked, it gave me some weird UI things. When I am typing, ti starts to give me suggestions from those list items that are already on the board, which was annoying. This does not seem to be the case with `noModelOptions`.

Reply:
Yeah that's one of the differences.

Reply:
> I added `name` attribute to the input tag. Though it worked, it gave me some weird UI things. When I am typing, ti starts to give me suggestions from those list items that are already on the board, which was annoying. This does not seem to be the case with `noModelOptions`.

I think what you're referring to is the default behavior of web browsers in regards to already filled form elements. I don't think bypassing this default behavior with `[ngModelOptions]` is a good choice, since it doesn't comply with [accessibility standards](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0#qr-ensure-compat-rsv). IMO you should bring back the `name` attribute.

Reply:
Hey do you mind making an issue for this? Thank you so much for calling this out.